src/gentropy/dataset/l2g_prediction.py

```diff
@@ -83,3 +85,36 @@ def from_credible_set(
         )
 
         return l2g_model.predict(fm, session)
+
+    def to_disease_target_evidence(
+        self: L2GPrediction,
+        study_locus: StudyLocus,
+        study_index: StudyIndex,
+        l2g_threshold: float = 0.05,
+    ) -> DataFrame:
```

Comment:
Virtually this is another source of evidence generation, and it is not going to go through validation unlike the others.
Maybe it's annoying, but I think it'd be nice to include a new data model.

Reply:
I was thinking about this... and not convinced. Evidence doesn't have any downstream application, no methods are called on evidence dataset... Especially considering we only have five or so columns, of which two are constant. I'm quite inclined not to have an other dataset. Later maybe... eg. in case if the generation of the VEP input requires it.